src/components/PokeCard.js

```diff
@@ -0,0 +1,44 @@
+import './PokeCard.css';
+import React, { useState } from 'react';
+import { mockPokemonData } from '../mock/pokeData';
+
+function getData() {
+    return Promise.resolve( mockPokemonData );
+}
+
+const initialPokemonState = {
+    name: '',
+    sprites: {
+        front_default: '',
+        front_shiny: ''
+    },
+    video: ''
+};
+
+function PokeCard() {
+    const [pokemon, setPokemon] = useState(initialPokemonState);
+
+    getData()
```

Comment:
now I see why you're not passing values over the component.  Even though, you should be able to keep components agnostic of the data source.  So, a single component in charge of requesting data and validating the status of that transaction, then the children components in charge of painting that information only

Reply:
Thank you, this review is very insightful to me! I came from Angular where components don't make request, and I'm just getting used to the React way.